Dockerfile

```diff
@@ -1,7 +1,28 @@
-FROM node:10
-RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 && chmod +x /usr/local/bin/dumb-init
-WORKDIR /app
-COPY . .
-RUN npm i --production
-ENTRYPOINT ["/usr/local/bin/dumb-init", "node", "src/server/bin.js"]
+FROM node:lts-buster
 
+# Install deps
+RUN apt-get update && apt-get install -y
+
+# Setup directories for the `node` user
+RUN mkdir -p /home/node/app/stardust/node_modules && chown -R node:node /home/node/app/stardust
+
+WORKDIR /home/node/app/stardust
+
+# Install node modules
+COPY package.json ./
+# Switch to the node user for installation
+USER node
+RUN npm install --production
+
+# Copy over source files under the node user
+COPY --chown=node:node ./src ./src
+COPY --chown=node:node ./README.md ./
+
+# stardust defaults to 5892
+EXPOSE 5892
+
+# Available overrides (defaults shown):
+#   --disableMetrics=false
+# Server logging can be enabled via the DEBUG environment variable:
+#   DEBUG=signalling-server,signalling-server:error
+CMD [ "node", "src/server/bin.js"]
```

Comment:
passing the signal doesn't work too well without dumb-init, so you can't quit it properly if running interactivly

Reply:
what do you recommend here?

Reply:
use the dumb-init as before? I

Reply:
re-adding the wget line and launching the app via dumb-init again